Dockerfile.compute-node

```diff
@@ -89,14 +89,20 @@ COPY --from=neon-pg-ext-build --chown=postgres /usr/local/pgsql /usr/local
 COPY --from=compute-tools --chown=postgres /home/nonroot/target/release/compute_ctl /usr/local/bin/compute_ctl
 
 RUN apt update &&  \
-    apt install -y libreadline-dev libossp-uuid-dev gdal-bin libgdal-dev libprotobuf-c-dev && \
+    apt install -y \
+        libreadline8 \
+        libossp-uuid16 \
+        libgeos-c1v5 \
+        libgdal28 \
+        libproj19 \
+        libprotobuf-c1 && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
Some comments would be nice on this step. If I understand correctly, this step installs the libraries that are needed by PostgreSQL, and the extensions we now install.

It would also be nice to somehow mark which libraries are needed by which extension.